api/comment/comment.router.js

```diff
@@ -3,16 +3,55 @@ const router = express.Router();
 const Post = require('../models/Post');
 const Comment = require('../models/comment.schema');
 
+//get all comments
+router.get('/', async (req, res) => {
+    try {
+        // return all the comments or can make a limit
+        const comments = await Comment.find();
+        res.json(comments);
+    } catch (err) {
+        res.json({ message: err });
+    }
+});
+
+
 //get one comment
-router.get('/:comment_id', (req, res) => {
-    //returns an empty array
-    res.json([]);
-})
+router.get('/:comment_id', async (req, res) => {
+    try {
+        const comment = await Comment.findById(req.params.comment_id);
+        // .populate('comments');
```

Comment:
comments are not populated in this route. Please remove the comment to keep the code as simple and readable as possible for everyone